models/articles.model.js

```diff
@@ -15,7 +16,11 @@ exports.readArticleById = (article_id) => {
     return rows[0];
   });
 };
-exports.readAllArticles = (sort_by = "created_at", order = "DESC") => {
+exports.readAllArticles = async (
```

Comment:
Nice getting async/await in here. My advice would be to be consistent through your app, with both models and controllers. Either go with async/await or promises syntax. You of course have time to refactor this in the coming weeks! 😄